src/utl/include/utl/Logger.h

```diff
@@ -49,6 +50,7 @@ class Progress;
   X(GRT)                \
   X(GUI)                \
   X(IFP)                \
+  X(LEXT)               \
```

Comment:
Module names should be three letters as the logger formatting assumes that.  I find the name logic extraction not very descriptive so let's discuss before you spend time to change it.